src/index.js

```diff
@@ -45,6 +45,15 @@ export default class ESjs {
     this.indexDoc(doc);
   }
 
+  removeDoc(id) {
```

Comment:
Open to other Names for this, since we're not passing a doc as a parameter, just the id.  Maybe this should be called `removeDocById`?